cac-test-local.yml

```diff
@@ -178,6 +178,9 @@ unclassified:
                 credentialsId: hmcts-jenkins-cnp
                 repoOwner: hmcts
                 repository: cnp-jenkins-library
+  defaultFolderConfiguration:
+    healthMetrics:
+    - "primaryBranchHealthMetric"
```

Comment:
no more moaning about unhealthy folders for some bad PRs, take health from master